src/main/java/seedu/address/logic/commands/edit/EditNoteCommand.java

```diff
@@ -68,15 +68,14 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_ITEM);
         }
 
-        return new EditCommandResult(editedNote.toString(),
-                String.format(MESSAGE_EDIT_NOTE_SUCCESS, editedNote), model.getDisplayType());
+        return new EditCommandResult(editedNote.toString(), MESSAGE_EDIT_NOTE_SUCCESS, model.getDisplayType());
     }
 
     /**
      * Create a new Note after it has been edited.
-     * @param toEdit
-     * @param editNoteDescriptor
-     * @return
+     * @param toEdit the note that will be edited.
```

Comment:
I'm not sure of the convention for writing Javadoc, i.e. capitalisation and punctuation.

We may need to look up and standardise across.